src/mumble/DBus.cpp

```diff
@@ -84,6 +84,34 @@ void MumbleDBus::focus() {
 	g.mw->activateWindow();
 }
 
+/// Change when Mumble transmits voice.
+///
+/// @param mode The new transmit mode (0 = continous, 1 = voice activity, 2 = push-to-talk)
```

Comment:
Sorry I forgot to tell you that we keep the documentation with the declaration (in the  header file) and not the implementation :see_no_evil: